SAMtags.tex

```diff
@@ -234,13 +239,39 @@ \subsection{Barcodes}
 
 \begin{description}
 \item[BC:Z:\tagvalue{sequence}]
-Barcode sequence, with any quality scores stored in the {\tt QT} tag.
-
-\item[QT:Z:\tagvalue{qualities}]
-Phred quality of the barcode sequence in the {\tt BC} (or {\tt RT}) tag. Same encoding as {\sf QUAL}.
+Barcode sequence (Identifying the sample/library), with any quality scores (optionally) stored in the {\tt QT} tag.
+The {\tt BC} tag should match the {\tt QT} tag in length. 
+In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the barcodes and qualities and places a dash (`{\tt -}') between the barcodes from the same template. 
```

Comment:
My point is that we state we string base barcodes together with dashes but not the qualities, and then also state the BC and QT must be the same length.   I understand the intention - that length here is in number of base calls and not length of string, but it is ambiguous.  I don't mean that we *should* have different lengths.

Specifically it is this sentence: "The {\tt BC} tag should match the {\tt QT} tag in length. In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the barcodes and qualities and places a dash (`{\tt -}') between the barcodes from the same template."

There is no corresponding statement in QT that it will also have dashes between barcodes from the same template.  Adding them here may also be problematic as dash is a valid quality (unlike a valid base-call).  We need to be explicit - does QT also have the dashes?  If so state this.  If not we should rephrase the comment about BC and QT being the same length to something like "The number of nucleotides in the {\tt BC} tag should match the length of the {\tt QT} tag."